blob/blob.go

```diff
@@ -16,42 +19,70 @@ import (
 	"github.com/celestiaorg/celestia-node/share"
 )
 
+//nolint:unused
```

Comment:
Weird that its still here, even there was a comment addressing it

Reply:
https://github.com/celestiaorg/celestia-node/pull/3610#discussion_r1700202243

Reply:
That comment referred to a diff code and path tho

Reply:
Although, it doesn't hurt to keep it if we keep some other deadcode anyway

Reply:
> That comment referred to a diff code and path tho

yes, but that's the only code using this. So the linter also detects this as unused.